pyproject.toml

```diff
@@ -45,6 +46,7 @@ dev = [
     "sphinx",
     "sphinx-pyproject",
     "autodoc_pydantic",
+    "myst-parser",
```

Comment:
We can split the dependencies to what's needed for the project itself, and for development purposes.

```toml
dependencies = [
    "epydemix",
    "pandas",
    "numpy",
    "scipy",
    "holidays",
    "matplotlib",
    "pydantic",
    "pyyaml",
    "psutil",
    "sodapy",
    "epiweeks",
]

[project.optional-dependencies]
dev = [
    "sphinx",
    "sphinx-pyproject",
    "autodoc_pydantic",
    "myst-parser",
]
```